src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -128,6 +128,8 @@ def set_tags(scope: Scope, result: JSONData) -> None:
         scope.set_tag(
             "stacktrace_link.auto_derived", result["config"]["automaticallyGenerated"] is True
         )
+    if result["codecov"].get("attemptedUrl"):
```

Comment:
attempted_url

Reply:
It seems that keys returned in the API have been following camel case syntax rather than snake case.
We can stick with camel case for API responses unless we would prefer to change it (I would but I don't prefer it strongly).

Reply:
ah I misread what that line was doing, camel case is correct!